source/_integrations/switch_as_x.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: Switch as X
+title: Change device type of a switch
```

Comment:
> * UI string has been renamed in core, see
>   [home-assistant/core@`f205d50`/homeassistant/components/switch_as_x/strings.json#L2](https://github.com/home-assistant/core/blob/f205d50ac710cc4cad505295fb72c5a3f84603fd/homeassistant/components/switch_as_x/strings.json#L2)

That is not what the docs use as the source: 

<https://github.com/home-assistant/core/blob/f205d50ac710cc4cad505295fb72c5a3f84603fd/homeassistant/components/switch_as_x/manifest.json>

The next release this change will be automatically overwritten.

../Frenck